### PR TITLE
Remove layout package for the SDS repo

### PR DESCRIPTION
**Pull request by yerramshilpa**
## Type of Change (Select One and Apply Github Label)

- [ ] Bug fix (non-breaking change which fixes an issue) -> Apply bugfix label
- [ ] New feature (non-breaking change which adds functionality) -> Apply enhancement label
- [ ] Breaking change (fix or feature that would cause existing functionality to change) -> Apply breaking label

## Which browsers have you tested?
- [ ] Internet Explorer 11
- [ ] Edge
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

## Checklist:

- [ ] I have read the (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md] document.
- [ ] My code passes the automated linter.
- [ ] This code has been reviewed by another team member and passes the reviewer checklist found in (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md]
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code is 508 compliant as tested by AMP and JAWS
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: apps/sam-design-system-site/tsconfig.json [hunk 1]
```diff
@@ -31,13 +27,20 @@
...
+    },
+    {
+      "path": "libs/packages/sam-material-extensions"
+    },
+    {
+      "path": "libs/packages/sam-formly"
+    },
+    {
+      "path": "libs/packages/layouts"
+    }
```
Comment (Bibek-Regmi) [inline apps/sam-design-system-site/tsconfig.json:41]: can we remove the reference to layouts here?

**Comment by Bibek-Regmi** [comment]
We should also have links for layouts in app.component.html. I don't see the updates to that file in this PR

File: libs/documentation/src/lib/pages/introduction/introduction.component.ts [hunk 2]
```diff
@@ -1,18 +1,17 @@
 import { Component, OnInit, ViewEncapsulation } from '@angular/core';
```
Comment (Bibek-Regmi) [inline libs/documentation/src/lib/pages/introduction/introduction.component.ts:1]: Could you also check the html file for this component for references to layouts?

**Comment by Bibek-Regmi** [comment]
Federalist build issue seems be be caused by references to layouts in package.json's scripts

File: libs/documentation/src/lib/pages/introduction/introduction.component.ts [hunk 3]
```diff
@@ -1,18 +1,17 @@
...
 })
-
 export class IntroductionComponent implements OnInit {
   packages: any = {};
 
   constructor() {
     this.packages.components = require('./../../../../../packages/components/package.json');
     this.packages.formly = require('./../../../../../packages/sam-formly/package.json');
-    this.packages.layouts = require('./../../../../../packages/layouts/package.json');
+    // this.packages.layouts = require('./../../../../../packages/layouts/package.json');
```
[the remark this answers is not in the document]
Comment (yerramshilpa) [reply libs/documentation/src/lib/pages/introduction/introduction.component.ts]: Removed

File: libs/documentation/src/lib/components/workspace-tier-2/item/item.module.ts [hunk 4]
```diff
@@ -12,7 +12,6 @@ import { ComponentWrapperComponent } from './../../../shared/component-wrapper/c
 import { ResultsModule } from './demos/results/results.module';
 import { ResultsComponent } from './demos/results/results.component';
 
-
 declare var require: any;
```
Comment (Bibek-Regmi) [inline libs/documentation/src/lib/components/workspace-tier-2/item/item.module.ts:15]: This file seems like it was created for layouts demo - can we delete this or is there value in keeping it?

File: libs/documentation/src/lib/components/landing/link/link.module.ts [hunk 5]
```diff
@@ -34,7 +34,7 @@ export const ROUTES = [
       title: 'Landing / Link',
       items: [
         {
-          pkg: 'layouts',
```
Comment (Bibek-Regmi) [inline libs/documentation/src/lib/components/landing/link/link.module.ts:37]: It seems like quite a few of these files where package is layouts can be removed as well

Comment (yerramshilpa) [reply libs/documentation/src/lib/components/landing/link/link.module.ts:37]: Removed un related demo's from the app

File: libs/documentation/src/lib/pages/introduction/introduction.component.ts (hunk 2 above)
Comment (yerramshilpa) [reply libs/documentation/src/lib/pages/introduction/introduction.component.ts:1]: fixed and removed all references

**Review by Bibek-Regmi — APPROVED** [review]

**Pull request merged**